### ISSUE TITLE
Missing icons for Checkstyle in IntelliJ

### ISSUE THREAD

**Issue by mkapra** [mallowigi/a-file-icon-idea#93]
I don't know if this is the right project for my issue but I have a problem with missing icons on IntelliJ.
I have installed this plugin: [https://plugins.jetbrains.com/plugin/1065-checkstyle-idea](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea)

There are missing some Icons for the Checkstyle Warning Checks.

![Screenshot 2020-06-15 at 13 13 54](https://user-images.githubusercontent.com/34742358/84651482-52afad00-af0a-11ea-922e-4ed484f8a6c4.png)

**Comment by mallowigi** [comment]
Yes, there's a big issue with the IconLoader, so far I couldn't find a solution.

**Comment by Frederick888** [comment]
@mallowigi May I know whether this is an IntelliJ issue? If so is there an existing ticket at https://youtrack.jetbrains.com?

**Comment by mallowigi** [comment]
Haven't found the fix for this issue unfortunately, but I found a workaround by adding a button that will manually reload the icons, therefore allowing the icons to be rerendered. 

Will be available shortly.
